### DISCUSSION TITLE
DC5V ESP32-Cam Wifi Bluetooth Development Board Module OV2640 Camera ESP32 V3 Generation Chip Type-C Interface

### DISCUSSION THREAD

**Discussion by FBMinis** [jameszah/ESP32-CAM-Video-Telegram#62] [General]
This seems to be a new version of our beloved ESP32-CAM. The form factor is quite attractive...

https://www.aliexpress.com/item/1005007908129425.html

![Screenshot 2024-11-26 at 19-37-20 Dc5v ESP32-Cam wifi bluetooth módulo de placa de desenvolvimento ov2640 câmera esp32 v3 geração chip tipo-c interface - AliExpress 502](https://github.com/user-attachments/assets/cd48e44c-cc49-44ff-b375-eaf34ea1d2b2)

**Comment by jameszah** [comment]
Nice - i got a an s3 wroom with 2 usb-c and 8MB psram - but haven't experimented with it much.  Lots of gpio's so the struggle with pins is avoided.

That one says "v3" but not clear if it is esp32-s3 ??  I suppose one could figure it out from the specs.  That store has some tradition no-usb modules where they they do say "s3" -- no a little confusing.  But only $7 shipping to arrive dec 12, which is better than usual aliexpress.

I like the un-soldered pins as well!

**Reply by FBMinis** [reply]
The chip seems to have this written on it: https://www.tme.eu/en/details/esp32-d0wd-v3/iot-wifi-bluetooth-modules/espressif/
